micro-ui/web/micro-ui-internals/packages/modules/workbench/src/components/DigitJSONForm.js

```
@@ -255,111 +233,221 @@ const DigitJSONForm = ({
+      enabled: false, // We'll call mutateAsync manually
+    },
+  };
+  const secondFormatLocalizationMutation = Digit.Hooks.useCustomAPIMutationHook(reqCriteriaSecondUpsert);
+
+  const onSubmitV2 = async ({ formData }, e) => {
+    console.log("additional data", additionalProperties);
+
+    const transformedFormData = { ...formData };
+
+    for (const fieldName in additionalProperties) {
+      if (additionalProperties.hasOwnProperty(fieldName)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)